docker-compose.yml

```diff
@@ -142,7 +142,7 @@ services:
 #      - .:/modyn_host
     container_name: supervisor
     ports:
-      - 3000:50062
+      - "3000:50063"
```

Comment:
50062 is never the port for supervisor... I was not sure how other people correctly ran Modyn before